packages/cozy-harvest-lib/test/connections/triggers.spec.js

```diff
@@ -88,10 +88,14 @@ describe('Trigger mutations', () => {
     const longLoginResponseTime = 150
     const jobSuccessResponseTime = 100
 
+    const updatedJob = {
+      state: 'done'
+    }
+
     beforeAll(() => {
       // Mock realtime to respond at 100ms
       realtime.subscribe.mockImplementation(() => ({
-        onUpdate: fn => setTimeout(() => fn(), jobSuccessResponseTime)
+        onUpdate: fn => setTimeout(() => fn(updatedJob), jobSuccessResponseTime)
       }))
     })
```

Comment:
Can you add a test showing that if the state is not done or errored, the promise is not resolved ?